izpack-dist/pom.xml

```diff
@@ -69,6 +65,9 @@
                             <goal>copy-resources</goal>
                         </goals>
                         <configuration>
+                            <nonFilteredFileExtensions>
+                                <nonFilteredFileExtension>jar</nonFilteredFileExtension>
```

Comment:
@reinhapa Looking at [m-r-p](https://github.com/apache/maven-resources-plugin/blob/master/src/main/java/org/apache/maven/plugins/resources/ResourcesMojo.java#L175) the parameter name is ```nonFilteredFileExtensions```.